lib/InternalContactsFieldArray/UserField.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  Card,
+  Col,
+  KeyValue,
+  Layout,
+  Row,
+} from '@folio/stripes/components';
+import { AppIcon, Pluggable } from '@folio/stripes/core';
+
+import css from './UserField.css';
+
+export default class UserField extends React.Component {
+  static propTypes = {
+    id: PropTypes.string,
+    input: PropTypes.shape({
+      onChange: PropTypes.func.isRequired,
+      value: PropTypes.string,
+    }).isRequired,
+    meta: PropTypes.shape({
+      error: PropTypes.node,
+    }).isRequired,
+    onUserSelected: PropTypes.func.isRequired,
+    onUserUnselected: PropTypes.func.isRequired,
+    user: PropTypes.shape({
+      label: PropTypes.string,
+    }),
+  }
+
+  static defaultProps = {
+    user: {},
+  }
+
+  renderLinkUserButton = () => (
+    <Pluggable
+      aria-haspopup="true"
+      dataKey="user"
+      data-test-ic-link-user
+      disableRecordCreation
+      id={`${this.props.id}-search-button`}
+      marginBottom0
+      searchLabel={<FormattedMessage id="stripes-erm-components.contacts.addUser" />}
+      searchButtonStyle="primary"
+      selectUser={this.props.onUserSelected}
+      type="find-user"
+    >
+      <FormattedMessage id="stripes-erm-components.contacts.noUserPlugin" />
+    </Pluggable>
+  )
+
+  renderUnlinkUserButton = () => (
+    <Button
+      buttonStyle="danger"
+      data-test-ic-unlink-user
+      id={`${this.props.id}-unlink-button`}
+      marginBottom0
+      onClick={this.props.onUserUnselected}
+    >
+      <FormattedMessage id="stripes-erm-components.contacts.unlinkUser" />
+    </Button>
+  )
+
+
+  renderUser = () => {
+    const {
+      email = '-',
+      firstName = '-',
+      lastName = '-',
+      phone = '-',
+    } = get(this.props.user, 'personal', {});
+
+    return (
+      <div data-test-user-card>
+        <Row>
+          <Col xs={6} md={3}>
+            <KeyValue label="Last name">
```

Comment:
Assuming this is still WIP. Translations I mean.

Reply:
Yup yup. Good catch though :D